downloads/swan-lake-release-notes/swan-lake-alpha2/RELEASE_NOTE.md

```diff
@@ -2,41 +2,599 @@
+    - [For New Users](#for-new-users)
+- [Highlights](#highlights)
+- [What is new in Ballerina Swan Lake Alpha2](#what-is-new-in-ballerina-swan-lake-alpha2)
+    - [Language](#language)
+        -  [Support for Mapping and Error Binding Patterns in the Match Statement](#support-for-mapping-and-error-binding-patterns-in-the-match-statement)
+    - [Runtime](#runtime)
+        - [Support for Configurable Variables of Record and Table Types](#support-for-configurable-variables-of-record-and-table-types)
+        - [Support for Decrypting String Values Using the New Config Lang Library](#support-for-decrypting-string-values-using-the-new-config-lang-library)
+    - [Standard Library](#standard-library)
+        - [HTTP Module Improvements](#http-module-improvements)
+        - [MIME Module Improvements](#mime-module-improvements)
+        - [WebSocket Module Improvements](#websocket-module-improvements)
+        - [GraphQL Module Improvements](#graphql-module-improvements)
+        - [WebSub Module Improvements](#websub-module-improvements)
+        - [WebSubHub Module Improvements](#websubhub-module-improvements)
+        - [IO Module Improvements](#io-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [TCP Module Improvements](#tcp-module-improvements)
+        - [UDP Module Improvements](#udp-module-improvements)
+        - [Crypto Module Improvements](#crypto-module-improvements)
+        - [JWT Module Improvements](#jwt-module-improvements)
+    - [Code to Cloud](#code-to-cloud)
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Debugger](#debugger)
+        - [Ballerina Shell REPL - EXPERIMENTAL](#ballerina-shell-repl-experimental)
+    - [Breaking Changes](#breaking-changes)
 
 ### Updating Ballerina
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to Ballerina Swan Lake Alpha2 as follows.
 
-#### For existing users
+#### For Existing Users
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
 
-> $ ballerina dist update
+> `bal update`
 
-However, you need to use the following commands instead of the above if you have installed:
+If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Alpha2.
```

Comment:
Same concern as above regarding this

Reply:
When a user is reading this, might try out the command at the top first, only to find out it doesn't work and then, a few lines down only it mentions the compatible version info

Reply:
@keizer619

Reply:
Lets include another section for ballerina update for users who using older version than 0.8.14 as in download page

Reply:
https://github.com/ballerina-platform/ballerina-dev-website/pull/1845/files#r578960154

Reply:
@pubudu91